work_calendar.gemspec

```diff
@@ -27,10 +27,10 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'activesupport', '~>6.0'
-  spec.add_development_dependency 'bundler', '~> 2.0'
-  spec.add_development_dependency 'minitest', '~> 5.0'
-  spec.add_development_dependency 'rake', '~> 10.0'
-  spec.add_development_dependency 'rubocop', '~>0.71'
-  spec.add_development_dependency 'rubocop-performance', '~>1.3.0'
+  spec.add_development_dependency 'activesupport'
+  spec.add_development_dependency 'bundler'
+  spec.add_development_dependency 'minitest'
+  spec.add_development_dependency 'rake'
+  spec.add_development_dependency 'rubocop'
+  spec.add_development_dependency 'rubocop-performance'
```

Comment:
Perhaps I should set fixed versions after the upgrade, as in the `.lock` file?

Reply:
Does it matter though? We can keep versions unspecified so those will get updated during later changes.